tests/testthat/test-05-calibrate.R

```diff
@@ -147,6 +148,12 @@ test_that("can calibrate a model result", {
                  "anc_clients", "anc_plhiv", "anc_already_art",
                  "anc_art_new", "anc_known_pos",
                  "anc_tested_pos", "anc_tested_neg"))
+
+  ## Table metadata is returned
+  expect_equal(names(result$tableMetadata), "presets")
+  expect_length(result$tableMetadata$presets, 2)
+  expect_equal(names(result$tableMetadata$presets[[1]]),
+               c("label", "column", "area_id"))
```

Comment:
i believe that is why the build is failing
```suggestion
               c("label", "column", "row"))
```

Reply:
Thanks for that